src/main/java/hudson/plugins/logparser/LogParserPublisher.java

```diff
@@ -169,6 +169,9 @@ public void perform(Run<?, ?> build, FilePath workspace, Launcher launcher, Task
         public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
         private volatile ParserRuleFile[] parsingRulesGlobal = new ParserRuleFile[0];
         private boolean useLegacyFormatting = false;
+        private String colorWarning = "orange";
```

Comment:
Sorry I don't get this one, what are you calling *global* color selection? It's already in the descriptor class, I must be missing something here.